tests/functional/test_basic.py

```diff
@@ -253,6 +253,69 @@ def test_pywin32_win32ui(pyi_builder):
     pyi_builder.test_script('pyi_pywin32_win32ui.py')
 
 
+def test_ascii_path(pyi_builder):
+    distdir = pyi_builder._distdir
+    dd_ascii = distdir.encode('ascii', 'replace').decode('ascii')
+    if distdir != dd_ascii:
+        pytest.skip(reason="Default build path not ASCII, skipping...")
+
+    pyi_builder.test_script('pyi_path_encoding.py')
+
+@skipif_win
+def test_osx_linux_unicode_path(pyi_builder):
+    # Mac and Linux should handle 'unicode' type filenames without problem.
+    distdir = pyi_builder._distdir
+    unicode_filename = u'\u011b\u0161\u010d\u0159\u017e\u00fd\u00e1\u00ed\u00e9\u65e5\u672c\u8a9e'
```

Comment:
Good point. I was just being overly cautious with the `\u` escapes, making sure the unicode strings were what I wanted them to be in the code.